### PR TITLE
Some housekeeping, mostly update language list

### PR DESCRIPTION
**Pull request by birkenfeld**

### CODE REVIEW

File: CHANGES [hunk 1]
```diff
@@ -47,7 +47,7 @@ Version 2.5.0
 - Added lexers:
 
   * Email (PR#1246)
-  * Erlang, Elxir shells (PR#823, #1521)
+  * Erlang, Elixir shells (PR#823, #1521)
```
Comment (Anteru) [inline CHANGES]: Nice catch!

File: doc/languages.rst [hunk 2]
```diff
@@ -10,103 +10,167 @@ Programming languages
...
 * Objective-J
 * Octave
 * OCaml
+* Opa
+* OpenCOBOL
+* `ParaSail <http://www.parasail-lang.org>`_
+* Pawn
 * PHP
-* `Perl 5 <http://perl.org>`_ and `Perl 6 <https://perl6.org>`_
+* `Perl 5 <https://perl.org>`_ and `Perl 6 <https://perl6.org>`_
```
Comment (Anteru) [inline doc/languages.rst]: Perl6 is Raku now? Not sure if we care yet: https://raku.org/

File: doc/languages.rst [hunk 3]
```diff
@@ -115,55 +179,99 @@ Programming languages
 Template languages
 ------------------
 
+* Angular2 templates
```
Comment (Anteru) [inline doc/languages.rst]: Angular has no (meaningful) version number any more, should be `Angular templates`. Angular 2 was the big move from AngularJS to "Angular".

File: doc/languages.rst (hunk 2 above)
Comment (birkenfeld) [reply doc/languages.rst]: yeah, will fix

File: doc/languages.rst (hunk 3 above)
Comment (birkenfeld) [reply doc/languages.rst]: ok. I don't know anything about Javascript stuff :)

Comment (Anteru) [reply doc/languages.rst]: I unfortunately know more about this than I wanted to know :)

**Pull request merged**